### PR TITLE
publish topics integration guide

### PR DESCRIPTION
**Pull request by nancymic2**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:
- publish topics integration guide
- staged: https://pr-6483-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/topics/integration-guide/

### CODE REVIEW

**Comment by samdutton** [comment]
One small thing: I wonder if subhead could be more factual: 'Learn how to work with the API, from end to end.' is fine, but it doesn't really tell me what this doc is for.

Maybe something like 'Learn how to test the Topics API for ad tech use cases.' or 'Learn how to use the Topics API to meet ad tech use cases.'

**Comment by samdutton** [comment]
Also wondering if we could have an aside at the top, specifically asking for feedback. Something like this:

{% Aside %}
We'd love to know how this article could be improved! 

Follow the [Improve article](https://github.com/GoogleChrome/developer.chrome.com/blob/main/site/en/docs/privacy-sandbox/topics/integration-guide/index.md) link at the foot of the page to make suggestions.
{% endAside %}

Maybe... Not sure.

**Review by samdutton — APPROVED** [review]

File: site/en/docs/privacy-sandbox/topics/integration-guide/index.md [hunk 1]
```diff
@@ -0,0 +1,474 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Topics API integration guide'
+subhead: Learn how to use the Topics API to meet specific ad tech use cases.
+description: Learn how to use the Topics API to meet specific ad tech use cases.
+date: 2023-06-01
```
Comment (rachelandrew) [inline site/en/docs/privacy-sandbox/topics/integration-guide/index.md:6]:
```suggestion
date: 2023-06-02
```

File: site/en/docs/privacy-sandbox/topics/integration-guide/index.md [hunk 2]
```diff
@@ -0,0 +1,474 @@
...
+1. If you encounter unfamiliar terms, review the [Privacy Sandbox glossary](/docs/privacy-sandbox/glossary/).
+1. For more information on Chrome concepts, such as origin trials or Chrome flags, review the short videos and articles available at [goo.gle/cc](http://goo.gle/cc).
+
+
+## Build and test locally
+
+This section describes how to try out the Topics API as an individual developer. 
+[Participate in the origin trial](#participate-in-the-origin-trial) explains how to test at scale with your users by taking part in the origin trial.
+
+1. Local testing and deployment (Estimated time: ~2 days)
```
Comment (rachelandrew) [inline site/en/docs/privacy-sandbox/topics/integration-guide/index.md:68]:
```suggestion
1. Local testing and deployment (Estimated time: around 2 days)
```

Comment (rachelandrew) [reply site/en/docs/privacy-sandbox/topics/integration-guide/index.md:68]: Use words for things like this.

File: site/en/docs/privacy-sandbox/topics/integration-guide/index.md [hunk 3]
```diff
@@ -0,0 +1,474 @@
...
+
+You have more fine-grained control over Topics features by running Chrome from the command line. For example, it's possible to set Topics epochs (the time frame used by the API to calculate user interests) and configure the behavior of the API according to your needs.
+
+It's important to remember that if **chrome://flags/#privacy-sandbox-ads-apis** is enabled, this will override your command line epoch setting, returning it to the default value (currently one week).
+
+### Preview Topics API mechanics
+
+You can get visibility into the underlying Topics API mechanics locally by using the [chrome://topics-internals](/docs/privacy-sandbox/topics/#debug) tools. 
+
+{% Img src="image/RtQlPaM9wdhEJGVKR8boMPkWf443/4LysAisUZpkryMHwk18f.png", alt="Take a look inside the Topics API at chrome://topics-internals", width="768", height="669" %}
```
Comment (rachelandrew) [inline site/en/docs/privacy-sandbox/topics/integration-guide/index.md:90]:
```suggestion
{% Img src="image/RtQlPaM9wdhEJGVKR8boMPkWf443/4LysAisUZpkryMHwk18f.png", alt="Take a look inside the Topics API at chrome://topics-internals.", width="768", height="669" %}
```

File: site/en/docs/privacy-sandbox/topics/integration-guide/index.md [hunk 4]
```diff
@@ -0,0 +1,474 @@
...
+
+Use the Topics API Internals tool to locally test the classifier based on the sites you visit. 
+
+With this tool, you can review:
+
+* Topics State: Display topics observed for the current user.
+* Classifier: Preview topics inferred for hostnames.
+* Features and Parameters: View API parameter values to check that feature flags are working as intended.
```
Comment (rachelandrew) [inline site/en/docs/privacy-sandbox/topics/integration-guide/index.md:98]:
```suggestion
* **Topics State:** Display topics observed for the current user.
* **Classifier:** Preview topics inferred for hostnames.
* **Features and Parameters:** View API parameter values to check that feature flags are working as intended.
```

File: site/en/docs/privacy-sandbox/topics/integration-guide/index.md [hunk 5]
```diff
@@ -0,0 +1,474 @@
...
+Given that the origin trial token has to be generated for the same origin as the code, it is recommended   that publishers use a script owned by the ad tech. This way, multiple sites can include the same script (as shown in the example below) which provides a third-party token registered for the ad tech origin (https://adtech.com).
+{% endAside %}
+
+```html
+<script src="https://adtech.com/js/topics.js">
+```
+
+## Build and deploy
+
+1. Collect topics by observing users in production—not scaled yet (Estimated time: ~1 week)
```
Comment (rachelandrew) [inline site/en/docs/privacy-sandbox/topics/integration-guide/index.md:362]: The following set of sentences don't have periods at the end, can you add them?

File: site/en/docs/privacy-sandbox/topics/integration-guide/index.md [hunk 6]
```diff
@@ -0,0 +1,474 @@
...
+    1. Understand your options: [iframe & JavaScript](/docs/privacy-sandbox/topics/#access-topics) OR [HTTP headers](/docs/privacy-sandbox/topics/#use-headers-to-access-and-observe-topics)
+    1. Define the domain of the iframe
+    1. Build the JavaScript code, using the [demo app](https://topics-demo.glitch.me/) as a code reference — or implement the headers option
+    1. Deploy Topics to your controlled environment (some production sites)
+    1. Register for the origin trial
+    1. Add the Topics implementation to some target sites (no more than five sites at this time)
+    1. Functional testing and validation
+2. [Optional] Use Topics data as a contextual signal (with URLs, tags, etc.) (Estimated time: ~3 days)
```
Comment (rachelandrew) [inline site/en/docs/privacy-sandbox/topics/integration-guide/index.md:370]:
```suggestion
2. [Optional] Use Topics data as a contextual signal (with URLs, tags, etc.) (Estimated time: around 3 days)
```

File: site/en/docs/privacy-sandbox/topics/integration-guide/index.md [hunk 7]
```diff
@@ -0,0 +1,474 @@
...
+    1. HTTP errors on topics calls;
+1. Try to limit changes to your implementation during the initial three weeks.
+
+## Scale to production
+
+Section step-by-step summary:
+
+1. Scale the implementation (production).
+    1. Add the iframe to multiple publisher's websites.
+1. Process and use topics data (Estimated time: ~4 weeks).
```
Comment (rachelandrew) [inline site/en/docs/privacy-sandbox/topics/integration-guide/index.md:453]:
```suggestion
1. Process and use topics data (Estimated time: around 4 weeks).
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]
Just a few nits in this. Missing punctuation etc. Ping me when fixed.

**Comment by samdutton** [comment]
Added a DO NOT MERGE label pending review from Leeron (see Chat).

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**